src/test/groovy/org/prebid/server/functional/model/request/auction/ImpExt.groovy

```diff
@@ -26,7 +26,7 @@ class ImpExt {
     String all
     String skadn
     String general
-    String foo
+    Foo foo
```

Comment:
@marki1an There is no need to create a new object Foo with the field anyProperty. You already have ext.data with all required values for all types. Please create a method for the default object and use it instead.